From 27dee52dfb7ecea26c8b70f652f54605b27252e9 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Mon, 20 Oct 2008 14:52:16 +0200
Subject: [PATCH] go7007 - Add missing dependency on sound subsystem

The dependency on the sound system is missing for go7007 driver, which
resulted in missing symbols like
  drivers/built-in.o: In function `go7007_snd_remove':
  : undefined reference to `snd_card_disconnect'
  drivers/built-in.o: In function `go7007_snd_remove':
  : undefined reference to `snd_card_free_when_closed'
  ...

This patch adds the dependency on CONFIG_SND, and selects CONFIG_SND_PCM
properly.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Tested-by: Ingo Molnar <mingo@elte.hu>
Cc: Ross Cohen <rcohen@snurgle.org>
Cc: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/go7007/Kconfig |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/go7007/Kconfig b/drivers/staging/go7007/Kconfig
index 57a121c..593fdb7 100644
--- a/drivers/staging/go7007/Kconfig
+++ b/drivers/staging/go7007/Kconfig
@@ -1,10 +1,12 @@
 config VIDEO_GO7007
 	tristate "Go 7007 support"
 	depends on VIDEO_DEV && PCI && I2C && INPUT
+	depends on SND
 	select VIDEOBUF_DMA_SG
 	select VIDEO_IR
 	select VIDEO_TUNER
 	select VIDEO_TVEEPROM
+	select SND_PCM
 	select CRC32
 	default N
 	---help---
-- 
1.5.6.5

